.travis.yml

```diff
@@ -141,8 +138,8 @@ jobs:
     # INTERMITTENTLY ALLOWED FAILURES
```

Comment:
You'll want to restructure the file a bit, so that it doesn't even mention intermittently allowed failures, since after this change there won't be any.